src/rust/src/lib.rs

```diff
@@ -91,6 +99,13 @@ fn _rust(py: pyo3::Python<'_>, m: &pyo3::types::PyModule) -> pyo3::PyResult<()>
     crate::x509::ocsp_resp::add_to_module(ocsp_mod)?;
     m.add_submodule(ocsp_mod)?;
 
+    let openssl_mod = unsafe {
+        // let ptr = PyInit__openssl();
+        // pyo3::types::PyModule::from_owned_ptr(py, ptr)
+        make_cryptography_openssl_module();
+        pyo3::types::PyModule::import(py, "_openssl")?
```

Comment:
This also makes `import _openssl` work after importing cryptography, could this be a problem for end users?

Reply:
Probably not. Maybe we should name it `_cryptography_openssl` or something to ensure no one accidentally uses it?